internal/am/delete_transfer_test.go

```diff
@@ -0,0 +1,93 @@
+package am_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/go-logr/logr"
+	"go.artefactual.dev/tools/mockutil"
+	temporalsdk_activity "go.temporal.io/sdk/activity"
+	temporalsdk_testsuite "go.temporal.io/sdk/testsuite"
+	"go.uber.org/mock/gomock"
+	tfs "gotest.tools/v3/fs"
+
+	"github.com/artefactual-sdps/enduro/internal/am"
+	sftp_fake "github.com/artefactual-sdps/enduro/internal/sftp/fake"
+)
+
+func TestDeleteTransferActivity(t *testing.T) {
+	filename := "fake_bag"
+	td := tfs.NewDir(t, "enduro-delete-transfer-test",
+		tfs.WithFile(filename, "Testing 1-2-3!"),
+	)
+
+	type test struct {
+		name     string
+		params   am.DeleteTransferActivityParams
+		recorder func(*sftp_fake.MockClientMockRecorder, am.DeleteTransferActivityParams)
+	}
+	for _, tt := range []test{
+		{
+			name: "Deletes transfer",
+			params: am.DeleteTransferActivityParams{
+				Destination: td.Path(),
+			},
+			recorder: func(m *sftp_fake.MockClientMockRecorder, params am.DeleteTransferActivityParams) {
+				m.Delete(
+					mockutil.Context(),
+					params.Destination,
+				).Return(nil)
+			},
+		},
+		{
+			name: "Logs when local file can't be read",
```

Comment:
This test and the following one aren't actually testing anything.  I can see a few choices to fix this:

1. Delete both tests
2. Use https://github.com/tonglil/buflogr (or similar) to record the log then assert that the actual value meets expectations
3. Return an error from the activity. This could be instead of logging or in addition to logging.

I actually prefer option #3.  I think returning an error will make the activity show up as "failed" in the Temporal UI, but we can not propagate the error so the workflow continues anyway.  

I know I said I *didn't* want the error to show up in the Temporal UI, but after thinking about it I think it is better to notify the operator that the delete failed.  I doubt anybody will be checking the app logs unless they know an error has occurred from checking the Temporal UI, the AM Dashboard, or from some other red flag.